src/tests/ftest/util/agent_utils.py

```diff
@@ -224,6 +227,53 @@ def start(self):
 
         super().start()
 
+    def dump_attachinfo(self):
+        """Run dump-attachinfo on the daos_agent."""
+        self.manager.job.set_sub_command("dump-attachinfo")
+        self.manager.job.sudo = True
+        self.attachinfo = run_pcmd(self.hosts,
+                                   str(self.manager.job))[0]["stdout"]
+        self.log.info("Agent attachinfo: %s", self.attachinfo)
+
+    def get_attachinfo_file(self):
```

Comment:
(pylint-inconsistent-return-statements) Either all return statements in a function should return an expression, or none of them should.